docs/bsp.md

```diff
@@ -165,12 +171,12 @@ trait BuildTarget {
   def data: Option[Json] // Note, matches `any` in the LSP.
 }
 
-object BuildTargetKind {
-  final val Library = 1
-  final val Test = 2
-  final val App = 3
-  final val IntegrationTest = 4
-  final val Bench = 5
+object BuildTargetTag {
+  val Library = "library"
+  val Test = "test"
+  val Application = "application"
+  val IntegrationTest = "integration-test"
+  val Benchmark = "benchmark"
 }
```

Comment:
I'm not sure, I think this can be left for IDEs to define. For example, you might want to use `no-intellij` instead to explicitly exclude IntelliJ only.

Reply:
I think there's value in having a standard way in BSP to say "there's no point in showing this target to users (because it's an implementation detail of the build", but it should just be a boolean flag somewhere, not a tag.

Reply:
Fair enough, I added `no-ide` but made it a tag so that clients can additionally respect client-specific exclusion tags like `no-intellij`. I've done this in the past for sbt projects that compile with Dotty or Scala Native.